snowexsql/api.py

```diff
@@ -10,14 +10,16 @@
 
 from snowexsql.conversions import query_to_geopandas, raster_to_rasterio
 from snowexsql.db import get_db
-from snowexsql.tables import ImageData, LayerData, PointData
+from snowexsql.tables import ImageData, LayerData, PointData, Instrument, \
+    Observer, Site, Campaign
+from snowexsql.tables.layer_data import LayerObservers
+from snowexsql.tables.point_data import PointObservers
```

Comment:
Line 15 and 16 look like unused imports?

This reminds me that we should really get setup on some basic code quality checks as documented in #107